dnssec.go

```diff
@@ -522,15 +524,15 @@ func (rr *RRSIG) sigBuf() []byte {
 }
 
 // publicKeyRSA returns the RSA public key from a DNSKEY record.
-func (k *DNSKEY) publicKeyRSA() *rsa.PublicKey {
+func (k *DNSKEY) publicKeyRSA() (*rsa.PublicKey, error) {
```

Comment:
It doesn't make much sense to add an `error` here if it's actual value is just ignored at every call site. I'd say keep this consistent with other similar functions and return `nil` on error.